static/skin/index.js

```diff
@@ -21,6 +23,23 @@
         return (url);
     }
 
+    function setCookie(cookieName, cookieValue) {
+        const date = new Date();
+        date.setTime(date.getTime() + ( 24 * 60 * 60 * 1000));
```

Comment:
space incoherence

Reply:
It would be nice to have a comment telling that `24*60*60*1000` is a day.
And even better, having a const name "1daydelta", computed once and doing the job of the comment.